client/src/app/pages/dependencies/dependencies.tsx

```diff
@@ -34,24 +35,66 @@ import {
 import { useFetchDependencies } from "@app/queries/dependencies";
 import { useSelectionState } from "@migtools/lib-ui";
 import { getHubRequestParams } from "@app/shared/hooks/table-controls";
-import { PageDrawerContent } from "@app/shared/page-drawer-context";
+import { DependencyAppsDetailDrawer } from "./dependency-apps-detail-drawer";
+import { useSharedAffectedApplicationFilterCategories } from "../issues/helpers";
 
 export const Dependencies: React.FC = () => {
   const { t } = useTranslation();
 
+  const allAffectedApplicationsFilterCategories =
+    useSharedAffectedApplicationFilterCategories();
```

Comment:
Changed this shared hook name since we are using it in dependencies. Open to name suggestions if this one is confusing.

Reply:
This name works for me, I can't really think of anything better than "affected application filter categories" for "filters on application properties that aren't for the app inventory page" 🙃